tests/integration/test_cli_service_accounts.py

```diff
@@ -91,9 +129,8 @@ def multiple_namespaces_and_service_accounts():
 
 
 @pytest.mark.parametrize("backend", VALID_BACKENDS)
-@pytest.mark.parametrize("action, resource", parameterize(ALLOWED_PERMISSIONS))
```

Comment:
Now that we have removed the use of `parameterize` function, we might as well clean it up. (since it is not referenced anymore)

Reply:
Done